rct229/ruletest_engine/ruletest_jsons/section21/rule_21_6.json

```diff
@@ -0,0 +1,395 @@
+                                            }
+                                        ],
+                                        "heating_ventilation_air_conditioning_systems": [
+                                            {
+                                                "id": "System 7",
+                                                "heating_system": {
+                                                    "id": "HHW Coil 1",
+                                                    "heating_system_type": "FLUID_LOOP",
+                                                    "hot_water_loop": "Boiler Loop 1"
+                                                },
+                                                "preheat_system": {
+                                                    "id": "Preheat Coil 1",
+                                                    "heating_system_type": "FLUID_LOOP",
+                                                    "hot_water_loop": "Boiler Loop 1"
+                                                },
+                                                "fan_system": {
+                                                    "id": "VAV Fan System 1",
+                                                    "fan_control": "VARIABLE_SPEED_DRIVE",
+                                                    "supply_fans": [
+                                                        {
+                                                            "id": "Supply Fan 1"
+                                                        }
+                                                    ],
+                                                    "return_fans": [
+                                                        {
+                                                            "id": "Return Fan 1"
+                                                        }
+                                                    ]
+                                                }
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "boilers": [
+                            {
+                                "id": "Boiler 1",
+                                "loop": "Boiler Loop 1",
+                                "design_capacity": 117228.44444444445,
+                                "operation_lower_limit": 0,
+                                "operation_upper_limit": 40000
+                            },
+                            {
+                                "id": "Boiler 2",
+                                "loop": "Boiler Loop 1",
+                                "design_capacity": 10257.488888888889,
+                                "operation_lower_limit": 35000,
+                                "operation_upper_limit": 70000
+                            }
```

Comment:
The Pass logic is
```python
(
                boiler_1_operation_lower_limit == 0
                and boiler_1_operation_upper_limit == boiler_1_rated_capacity
                and boiler_2_operation_lower_limit == boiler_2_rated_capacity
                and boiler_2_operation_upper_limit == boiler_2_rated_capacity * 2
            ) or (
                boiler_2_operation_lower_limit == 0
                and boiler_2_operation_upper_limit == boiler_2_rated_capacity
                and boiler_1_operation_lower_limit == boiler_1_rated_capacity
                and boiler_1_operation_upper_limit == boiler_1_rated_capacity * 2
            )
```

In this case, the `rated capacity` for boiler 1 shall be `40000` and `rated_capacity` for boiler 2 shall be `35000`